### ISSUE TITLE
target lists suggestion

### ISSUE THREAD

**Issue by matteoraggi** [salesagility/SuiteCRM#3313]
I suggest to show in the target lists the number of target subscribed into the target lists:
SUITECRM/clientis/index.php?module=ProspectLists&action=index&return_module=ProspectLists&return_action=index
![2017-03-24 14_58_45-eforte](https://cloud.githubusercontent.com/assets/14081756/24297371/d7427bf6-10a2-11e7-8e7e-678c2a163fb9.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [target lists suggestion](https://trello.com/c/wWcLggig/1-target-lists-suggestion)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [target lists suggestion](https://trello.com/c/Jxq0IgoQ/12-target-lists-suggestion)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
